src/main.rs

```diff
@@ -235,11 +235,20 @@ async fn main() -> anyhow::Result<()> {
     // If we're replaying the transaction, we need to sync to the previous block
     // and then replay all the transactions that happened in
     let transactions_to_replay = if let Command::ReplayTx(replay_tx) = command {
-        fork_details
+        match fork_details
             .as_ref()
             .unwrap()
             .get_earlier_transactions_in_same_block(replay_tx.tx)
-            .await
```

Comment:
why await is removed?

Reply:
Because `get_earlier_transactions_in_same_block` is no longer async (and that's because it doesn't need to call any async functions). Actually I think that happens in more places, but I don't know how to find them automatically...